docs/OV_Runtime_UG/ShapeInference.md

```diff
@@ -1,4 +1,6 @@
-# Changing input shapes {#openvino_docs_OV_UG_ShapeInference}
+# Changing Input Shapes {#openvino_docs_OV_UG_ShapeInference}
+
+This guide presents information about operating on Input Shapes, divided into C++ and Python sections.
```

Comment:
We don't need another introduction sentence here because we have a short Introduction section below. Instead, I would that suggest we remove the #Introduction heading, and integrate what will be introduced in this article into the introduction section. Besides, I remember we are supposed to change the global switcher for C++ and Python to be element-specific.